go-controller/pkg/cluster/cluster.go

```diff
@@ -98,3 +100,30 @@ func setupOVNMaster(nodeName string) error {
 	}
 	return nil
 }
+
+func validateOVNConfigEndpoint(ep *kapi.Endpoints) bool {
+	return len(ep.Subsets) == 1 && len(ep.Subsets[0].Ports) == 2 && len(ep.Subsets[0].Addresses) > 0
+}
+
+func extractDbRemotesFromEndpoint(ep *kapi.Endpoints) ([]string, string, string, error) {
```

Comment:
@numansiddique I know it's mostly just copied, but could we make the returned ports integers instead of string, and then carry that through right up until we need to exec something?

Reply:
I have done in ha_master, but not in config.UpdateOVNNodeAuth() in config.go.
If you want I can do it there too, but I would prefer a separate patch for that.

Reply:
@numansiddique separate patch is OK with me.